SmartDeviceLink/SDLClimateControlCapabilities.h

```diff
@@ -0,0 +1,113 @@
+//
+//  SDLClimateControlCapabilities.h
+//
+
+#import "SDLRPCMessage.h"
+#import "SDLDefrostZone.h"
+#import "SDLVentilationMode.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Contains information about a climate control module's capabilities.
+ */
+@interface SDLClimateControlCapabilities : SDLRPCStruct
+
+/**
+ * @abstractThe short friendly name of the climate control module.
```

Comment:
I don't believe we need the `@abstract` anymore in the latest versions of Xcode.